ccm_web/server/src/migrations/2021.06.16T19.36.19.create-user.ts

```diff
@@ -11,11 +11,13 @@ export const up: Migration = async ({ context: sequelize }) => {
     },
```

Comment:
Should we also specify `field` for single-word fields? It isn't necessary, but could make things clearer.

Reply:
I'd say no.

Django also has the ability to specify different names for objects in the DB, but I only use them when necessary.  So I would do the same here.

Reply:
K, thanks for the input